modules/thermal_hydraulics/src/components/FlowChannelBase.C

```diff
@@ -69,6 +69,9 @@ FlowChannelBase::validParams()
   params.addRequiredParam<std::string>("closures", "Closures type");
 
   params.addPrivateParam<std::string>("component_type", "pipe");
+
+  params.declareControllable("A f");
```

Comment:
Are you thinking that `f` could be controllable for things like uncertainty quantification? If so, I think that should not be done via control logic like this. There is this better design via materials we discussed earlier this year.

Reply:
I didn't make this decision - it's already controllable; see `Closures1PhaseBase::addWallFrictionFunctionMaterial`

Reply:
I guess, then it is fine to mark this as controllable.